packages/debug/service.go

```diff
@@ -52,12 +52,16 @@ func (s *defaultDebugModeService) Enable(ctx context.Context, req *pbMaintenance
 		return nil, createInternalError(fmt.Errorf("failed to activate maintenance mode: %w", err))
 	}
 
+	// Create new context because the admin dogu itself will be canceled
```

Comment:
It's a good thing that you added a comment because this is a fickle and rare situation.

I think the comment doesn't cut it properly. From earlier discussion I understand if the original context wasn't ignoring cancellation and which in turns ended the admin dogu's request process.

Does this comment suggestion add more clarity?
```suggestion
	// Create new context otherwise the cancellation of this context (f. i. by time-out) the admin dogu's request would be canceled as consequence.
```